mindjunkies/lecture/tests/test_views.py

```
@@ -97,28 +99,59 @@ def test_lecture_video_forbidden_for_non_enrolled(
+    }
+
+    response = client.post(url, video_data, follow=True)
+
+    assert response.status_code == 200
+    assert LectureVideo.objects.filter(lecture=lecture).exists()
+    messages = list(get_messages(response.wsgi_request))
+    assert any("uploaded successfully" in str(msg) for msg in messages)
+
+@pytest.mark.django_db
+def test_create_content_without_token(client):
+    user = baker.make(django_user_model)
```

Ruff (0.8.2)

Undefined name `django_user_model`

(F821)